apps/hash-frontend/src/pages/shared/entities-table.tsx

```diff
@@ -531,25 +555,33 @@ export const EntitiesTable: FunctionComponent<{
                 svg: {
                   transition: ({ transitions }) => transitions.create("color"),
                   color: ({ palette }) => palette.gray[50],
-                  fontSize: 18,
                 },
               },
             }}
           >
-            <ToggleButton disableRipple value="table" aria-label="table">
-              <Tooltip title="Table view" placement="top">
-                <Box sx={{ lineHeight: 0 }}>
-                  <ListRegularIcon />
-                </Box>
-              </Tooltip>
-            </ToggleButton>
-            <ToggleButton disableRipple value="graph" aria-label="graph">
-              <Tooltip title="Graph view" placement="top">
-                <Box sx={{ lineHeight: 0 }}>
-                  <ChartNetworkRegularIcon />
-                </Box>
-              </Tooltip>
-            </ToggleButton>
+            {(
+              [
+                "table",
+                ...(supportGridView ? (["grid"] as const) : []),
+                "graph",
+              ] satisfies EntityTableView[]
+            ).map((viewName) => (
+              <ToggleButton
+                key={viewName}
+                disableRipple
+                value={viewName}
+                aria-label={viewName}
+              >
+                <Tooltip
+                  title={`${viewName[0]?.toUpperCase()}${viewName.slice(1)} view`}
```

Comment:
This would be be simpler / easier to read if the values were already 'Table', 'Grid', 'Graph' – since it's just local state I think the simplicity is worth more than any ickiness of casing a value like that

Reply:
Agreed, have done so in https://github.com/hashintel/hash/pull/3974/commits/ef9e6a40582fae6880456923307149761c554639